tests/samplers/test_common.py

```diff
@@ -626,6 +627,76 @@ def test_wildcard_nested_in_wildcard(
         ps = [str(p) for p in islice(gen, len(expected))]
         assert ps == expected
 
+    @pytest.mark.parametrize("sampling_context", sampling_context_lazy_fixtures)
+    def test_wildcard_with_nested_variable(self, sampling_context: SamplingContext):
+        cmd = parse("${temp=cold}wearing __colors-${temp}__ suede shoes")
+        resolved_value =  str(next(sampling_context.generator_from_command(cmd)))
+        if isinstance(sampling_context.default_sampler, RandomSampler):
+            assert resolved_value == "wearing blue suede shoes" or resolved_value == "wearing green suede shoes"
```

Comment:
```suggestion
            assert resolved_value in ("wearing blue suede shoes", "wearing green suede shoes")
```

(repeated for the other instances of this same pattern 😄)

Reply:
Replaced with tuples.